frontend/packages/console-app/src/components/tour/tour-context.ts

```diff
@@ -107,12 +149,13 @@ export const useTourValuesForContext = (): TourContextType => {
   });
 
   useEffect(() => {
-    const { completed: initCompleted } = getTourLocalStorageForPerspective(activePerspective);
-    tourDispatch({ type: TourActions.initialize, payload: { completed: initCompleted } });
+    tourDispatch({ type: TourActions.initialize, payload: { completed } });
     setPerspective(activePerspective);
-  }, [activePerspective]);
+    // only run effect when the active perspective changes
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [activePerspective, loaded]);
```

Comment:
is `loaded` intentional here as only used as dependency ?

Reply:
Yes, loaded is intentional. If state doesn't exist for a perspective after the config map is loaded, this effect sets the default state for that perspective